sdk/client/src/listeners/first-load-listeners.tsx

```diff
@@ -258,25 +282,47 @@ export class FirstLoadListeners {
 				})
 
 			if (sThis.enableRecordingNetworkContents) {
-				resources = matchPerformanceTimingsWithRequestResponsePair(
-					resources,
+				httpResources = matchPerformanceTimingsWithRequestResponsePair(
+					httpResources,
 					sThis.xhrNetworkContents,
 					'xmlhttprequest',
 				)
-				resources = matchPerformanceTimingsWithRequestResponsePair(
-					resources,
+				httpResources = matchPerformanceTimingsWithRequestResponsePair(
+					httpResources,
 					sThis.fetchNetworkContents,
 					'fetch',
 				)
 			}
 		}
-		return resources
+
+		if (!sThis.disableRecordingWebSocketContents) {
+			webSocketResources = sThis.webSocketNetworkContents
+		}
+
+		return [...httpResources, ...webSocketResources]
```

Comment:
is it ok that the websocket resources are mixed with the http resources here? is that because this isn't for the messages themselves, but the overall websocket connection resources?

Reply:
Exactly, so these are just the higher level connection events that would be displayed intertwined in the network resources.